Makefile

```diff
@@ -600,6 +600,16 @@ upgrade_add_aggregator: ## Add Aggregator to Aligned Contracts
 	@echo "Adding Aggregator to Aligned Contracts..."
 	@. contracts/scripts/.env && . contracts/scripts/upgrade_add_aggregator_to_service_manager.sh
 
+set_aggregator_address: ## Add Aggregator to Aligned Contracts
```

Comment:
This should use NETWORK parameter as used in #1429 (https://github.com/yetanotherco/aligned_layer/pull/1429/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R567)

Reply:
addressed!